components/cart/cart.html

```diff
@@ -24,11 +24,19 @@
                 <input class="desktop-searchbar" type="text" placeholder="Search for products">
             </div>
             <div class="desktop-action-btn">
-                <div class="single-action-btn">
-                    <a href="">
-                        <svg class="right-nav-img" xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M12 5.9c1.16 0 2.1.94 2.1 2.1s-.94 2.1-2.1 2.1S9.9 9.16 9.9 8s.94-2.1 2.1-2.1m0 9c2.97 0 6.1 1.46 6.1 2.1v1.1H5.9V17c0-.64 3.13-2.1 6.1-2.1M12 4C9.79 4 8 5.79 8 8s1.79 4 4 4 4-1.79 4-4-1.79-4-4-4zm0 9c-2.67 0-8 1.34-8 4v3h16v-3c0-2.66-5.33-4-8-4z"/></svg>
-                    </a>
-                </div>
+                <div class="single-action-btn profile">
+                    <img class="avatar-round avatar-xsm small-profile" src="../assets/images/profile-avatar.jpg">
+                    <div class="profile-menu">
+                      <li class="profile-dropdown">My Profile</li>
+                      <li class="profile-dropdown">
+                          <a href="../cart/cart.html">Cart</a>
+                      </li>
+                      <li class="profile-dropdown">
+                          <a href="../wishlist/wishlist.html">Wishlist</a>
+                      </li>
+                      <li class="profile-dropdown">Logout</li>
+                    </div>
+                  </div>
                 <div class="single-action-btn">
                     <a href="../wishlist/wishlist.html">
                         <svg class="right-nav-img" xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M16.5 3c-1.74 0-3.41.81-4.5 2.09C10.91 3.81 9.24 3 7.5 3 4.42 3 2 5.42 2 8.5c0 3.78 3.4 6.86 8.55 11.54L12 21.35l1.45-1.32C18.6 15.36 22 12.28 22 8.5 22 5.42 19.58 3 16.5 3zm-4.4 15.55l-.1.1-.1-.1C7.14 14.24 4 11.39 4 8.5 4 6.5 5.5 5 7.5 5c1.54 0 3.04.99 3.57 2.36h1.87C13.46 5.99 14.96 5 16.5 5c2 0 3.5 1.5 3.5 3.5 0 2.89-3.14 5.74-7.9 10.05z"/></svg>
```

Comment:
Not related to this PR, but any reason why you are using svg tag directly here instead of have it from <img /> tag ?